packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseOpenAPIObject-test.js

```diff
@@ -47,6 +47,41 @@ describe('#parseOpenAPIObject', () => {
     expect(parseResult.api.get(0).href.toValue()).to.equal('/');
   });
 
+  it('can parse a document with servers array', () => {
+    const object = new namespace.elements.Object({
+      openapi: '3.0.0',
+      info: {
+        title: 'My API',
+        version: '1.0.0',
+      },
+      paths: {},
+      servers: [
+        {
+          url: 'https://user.server.com/1.0',
+        },
+        {
+          url: 'https://user.server.com/2.0',
+          description: 'The production API server',
+        },
+      ],
+    });
+
+    const parseResult = parse(context, object);
+
+    expect(parseResult.length).to.equal(1);
+    expect(parseResult.api.title.toValue()).to.equal('My API');
+    expect(parseResult.api.length).to.equal(1);
+
+    const serversArray = parseResult.api.get(0);
+    expect(serversArray).to.be.instanceof(namespace.elements.Category);
+    expect(serversArray.classes.toValue()).to.deep.equal(['hosts']);
+    expect(serversArray.length).to.equal(2);
+
+    const firstServer = serversArray.get(0);
+    expect(firstServer).to.be.instanceof(namespace.elements.Resource);
+    expect(firstServer.attributes.content[0].toValue().value).to.equal('https://user.server.com/1.0');
```

Comment:
I think you should be able to use the [`href`](https://github.com/apiaryio/api-elements.js/blob/ab47bebd0f421012f591982e1fe2512c41f6a9ab/packages/api-elements/lib/elements/Resource.js#L24) property on resource to simplify this code.

```suggestion
    expect(firstServer.href.toValue()).to.equal('https://user.server.com/1.0');
```

Reply:
changed